config/locales/fr.yml

```diff
@@ -475,9 +475,9 @@ fr:
         title: Paramètres de facturation
         invoice_style2?: Utiliser le modèle de facture alternatif qui détaille le montant de TVA agrégé par taux et l'information du taux de TVA par produit (pas adapté pour les instances affichant les prix HT)
         enable_receipt_printing?: Afficher les options d'impression de tickets de caisse dans le menu déroulant des commandes?
-  home:
+  shops:
```

Comment:
No worries, I will also update the translations on Transifex once the commit is merged to avoid any regression.